### DISCUSSION TITLE
No users listed on the client login screens

### DISCUSSION THREAD

**Discussion by gfreccia** [ltsp/ltsp#292] [Q&A]
Hi All,

is there a way to do not display all server users listed on the client login screens but just the box to write the username?

Thank you!

**Comment by alkisg** [comment]
Hi Giacomo, which display manager are you using? E.g. lightdm, sddm, gdm...

**Comment by gfreccia** [comment]
Hi Alkis,

I use the default settings, so I assume is Lightdm.

**Comment by alkisg** [comment]
Which desktop environment, MATE, GNOME, KDE, ...?
What's the output of this command?
`dpkg -l lightdm`

**Comment by gfreccia** [comment]
See attachment.
![Schermata del 2020-07-06 23-23-07](https://user-images.githubusercontent.com/35954627/86646928-f0653c00-bfdf-11ea-92f2-afde5f8d78b0.png)

I use Ubuntu Mate 20.04.

**Comment by alkisg** [comment]
Giacomo try this in ltsp.conf (remember to run `ltsp initrd` after changing ltsp.conf):

```shell
[clients]
LIGHTDM_CONF="greeter-hide-users=true"
```

```

**Comment by gfreccia** [comment]
Great Alkis, it works!
Thank you very much and Goodnight!
